src/components/molecules/Settings/WorkspaceList/WorkspaceCell/index.tsx

```diff
@@ -7,30 +7,30 @@ import { useT } from "@reearth/i18n";
 import { styled, useTheme } from "@reearth/theme";
 import { metricsSizes } from "@reearth/theme/metrics";
 
-import { Team as TeamType } from "../WorkspaceList";
+import { Workspace as WorkspaceType } from "../WorkspaceList";
 
-export type Team = TeamType;
+export type Workspace = WorkspaceType;
 
 export type Props = {
   className?: string;
-  team: Team;
+  workspace: Workspace;
   personal: boolean;
-  onSelect?: (t: Team) => void;
+  onSelect?: (t: Workspace) => void;
```

Comment:
t: Workspace doesn't make much sense. Please change it to at least `w: Workspace`, but preferably `workspace: Workspace`

Reply:
done